app/speakers/page.tsx

```diff
@@ -1,12 +1,73 @@
+import { Fragment } from 'react';
+import SpeakerCard, { TspeakerProps } from '../components/cards/SpeakerCard';
 import DFIHeader from '../components/common/DFIheader';
+import { Button } from '../components/ui/button';
+import SpeakerImg from '@/public/speakerimg.png';
+
+const SpeakersList: TspeakerProps[] = [
```

Comment:
We should move this to a seperate ts file or what do you think?
That way we can easily update speakers details from there when the details are avalable.